lib/utils/clipboard.dart

```diff
@@ -0,0 +1,173 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:fluffychat/presentation/model/clipboard/clipboard_image_info.dart';
+import 'package:flutter/services.dart';
+import 'package:matrix/matrix.dart';
+import 'package:mime/mime.dart';
+import 'package:super_clipboard/super_clipboard.dart';
+
+class Clipboard {
+  static final _clipboard = Clipboard._();
+
+  Clipboard._();
+
+  ClipboardReader? _reader;
+
+  static Clipboard get instance => _clipboard;
+
+  static const allImageFormatsSupported = [
```

Comment:
can we support all image file ? without detect image format ?

Reply:
any way, user still can paste file right ?

Reply:
just read it in bytes, No need to know its type

Reply:
yeah, but, i think it shouldn't because copy/paste is intented for small files, not large files. and telegram also did that, only allow copy images in chat not in preview